src/Groups/perm.jl

```diff
@@ -485,9 +484,11 @@ function ^(c::CycleType, e::Int)
   return t
 end
 
-function order(c::CycleType)
-  return reduce(lcm, map(x->fmpz(x[1]), c.s), init = fmpz(1))
-end
+order(::Type{T}, c::CycleType) where T = mapreduce(x->fmpz(x[1]), lcm, c.s, init = T(1))
```

Comment:
If the result shall have the type `T` then better call `T(x[1])` instead of `fmpz(x[1])`.

Reply:
Indeed, that was the intention and then I messed up *sigh*. Once again, thanks for your close attention